modules/nf-core/deepcell/mesmer/main.nf

```diff
@@ -0,0 +1,40 @@
+process DEEPCELL_MESMER {
+    tag "$meta.id"
+    label 'process_medium'
+
+    // Setting up the container to latest all the time because why not.
+    container "vanvalenlab/deepcell-applications:0.4.0"
+
+    // Mesmer, requieres one image to segment and the mpp(microns per pixel)
+    input:
+    tuple val(meta) , path(img)
+    tuple val(meta2), path(membrane_img)
+
+    // Output a .tif image, don't touch versions
+    output:
+    tuple val(meta), path("${prefix}.mask.tif"), emit: mask
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = "0.4.0"
+
+    """
+    python /usr/src/app/run_app.py mesmer \
```

Comment:
Does this not work with ` #!/usr/bin/python` (see [here](https://www.nextflow.io/docs/latest/process.html#scripts-a-la-carte)) ?

Reply:
It might, but is there any advantage / disadvantage of using either?